jurassic-journalists/classes.py

```diff
@@ -1,29 +1,29 @@
-#classes.py
 from PIL import ImageFont
-#letter class- each letter is one of these objects, and is rendered in order.
 class Letter:
+    """ letter class- each letter is one of these objects, and is rendered in order. """
     def __init__(self,char,size,font,color = (255,255,255,255),b=False,i=False,u=False):
+        """
+        char: character.
+        size: size of letter.
+        font: PIL truetype font object. TODO: add handling for other types
+        color: color of letter, RGBA tuple, range 0-1.
+        b: Bold flag.
+        i: Italics flag.
+        u: Underlined flag.
+        """
```

Comment:
For the future, are we going to use `typing` for params or do the docstrings like this?

Reply:
You can use both typing and doc string. `typing` is not used to describe a method rather used so what `type` of variable is being accepted. In here, doc string is used to describe what each variable means. http://google.github.io/styleguide/pyguide.html
![image](https://user-images.githubusercontent.com/39493678/72845769-4ed67200-3c6d-11ea-8a5c-a7b25b6730e1.png)

Reply:
yep just that the whole signature doesn't need to be explained when we just throw in the typehints like char and size above